### PR TITLE
Added github actions to automate build, test, and linting

### PR DESCRIPTION
**Pull request by dlabaj**
Added github actions to automate the build, test, and linting after a contributor checks in code.

This addresses Issue #15

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm run build
+    - run: npm run lint
+    - run: npm test
```
Comment (ssilvert) [inline .github/workflows/node.js.yml:30]: Should we be using yarn here instead of npm?

Comment (dlabaj) [reply .github/workflows/node.js.yml]: Done.

**Review by edewit — APPROVED** [review]

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - run: npm install -g yarn
+    - run: yarn
+    - run: yarn build
+    - run: yarn lint
```
Comment (edewit) [inline .github/workflows/node.js.yml:31]: maybe also add `yarn format`?

Comment (dlabaj) [reply .github/workflows/node.js.yml:31]: will do

**Pull request merged**